packages/components/psammead-media-player/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-media-player",
-  "version": "2.2.0",
+  "version": "2.2.1",
   "description": "Provides a media player with optional placeholder",
   "main": "dist/index.js",
   "module": "esm/index.js",
```

Comment:
You should also remove `psammead-assets` from the dependencies, since we're not using the SVG any longer.